processing/src/main/java/org/apache/carbondata/processing/loading/steps/InputProcessorStepWithNoConverterImpl.java

```diff
@@ -64,10 +63,13 @@
 
   private Map<Integer, GenericDataType> dataFieldsWithComplexDataType;
 
+  private short sdkUserCore;
```

Comment:
What does this variable mean? Please add comment, and maybe change to more meaningful name

Reply:
done.